script/Arcueid/Day 10/10_00_ARC10_2_2.txt

```diff
@@ -156,11 +158,12 @@ And so...
 [sha:31920d97c6a9151340749b5c7214c356b3849719]{
 -- Page 86, Offset 24475.
 -- だいたい、俺がおまえを嫌いだって言ったらどうする気なんだ？　見逃してくれるのか？」
-Besides, what would you do if I told you that I hated you? Would you have let me go?"
+Besides, what would you do if I had told you that I hated you? Would you have let me go?"
```

Comment:
what would you have done if I'd

Reply:
this is actually an italics candidate. Forgot to mark it.